test/reducers/context.spec.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+/* global describe, it */
+
+require('chai').should()
+const expect = require('chai').expect
+
+describe('context reducer', () => {
+  const { getContext, incrementContextResets } = require('../../actions/context')
+  const contextReducer = require('../../reducers/context')
+  it('should get context', () => {
```

Comment:
Given that the `default:` case of reducers just returns the state, seems like we really don't need the "get" actions at all.

Reply:
Yep, agreed, there's no need.

Reply:
Done